wicket-orientdb/src/main/java/ru/ydn/wicket/wicketorientdb/EmbeddOrientDbApplicationListener.java

```diff
@@ -74,7 +76,8 @@ else if(serverConfiguration!=null)
 			server.activate();
 			server.removeShutdownHook();
 			app.setServer(server);
-			app.getOrientDbSettings().setDatabasePoolFactory(server.getDatabasePoolFactory());
+
+			app.getOrientDbSettings().setDatabasePoolFactory(new ODatabasePoolFactory(server.getContext()));
```

Comment:
I believe that after requested changes by OrientDB - this creation of a factory should be done differently - through Orient.